Packs/VaronisDataSecurityPlatform/pack_metadata.json

```diff
@@ -5,7 +5,7 @@
     "currentVersion": "1.0.1",
     "author": "Varonis",
     "url": "",
-    "email": "dcohen@varonis.com",
+    "email": "https://www.varonis.com/support",
```

Comment:
Please add this url to the `url` field and add here your email

Reply:
@yaakovpraisler can we have only url and skip email?

Reply:
Yes, but please move the url to the `url` field

Reply:
@yaakovpraisler sure, done. Also I filled contribution form